searchlib/src/vespa/searchlib/attribute/reference_attribute.cpp

```diff
@@ -349,7 +340,9 @@ ReferenceAttribute::populateReferencedLids()
         saver.foreach_key([&store,&mapper,this](EntryRef ref)
                           {   const Reference &entry = store.get(ref);
                               entry.setLid(mapper.mapGidToLid(entry.gid()));
-                              _referenceMappings.syncReverseMappingIndices(entry); });
+                              _referenceMappings.syncReverseMappingIndices(entry);
```

Comment:
Consider creating a public syncMappings() on ReferenceMappings while making syncReverseMappingIndices() and syncForwardMapping() private.